unleash/src/main/java/com/silvercar/unleash/repository/ToggleBackupHandlerFile.kt

```diff
@@ -0,0 +1,86 @@
+package com.silvercar.unleash.repository
+
+import com.google.gson.JsonParseException
+import com.silvercar.unleash.UnleashException
+import com.silvercar.unleash.event.EventDispatcher
+import com.silvercar.unleash.event.UnleashEvent
+import com.silvercar.unleash.event.UnleashSubscriber
+import com.silvercar.unleash.util.UnleashConfig
+import org.apache.logging.log4j.LogManager
+import java.io.BufferedReader
+import java.io.FileNotFoundException
+import java.io.FileReader
+import java.io.FileWriter
+import java.io.IOException
+
+class ToggleBackupHandlerFile(config: UnleashConfig) : ToggleBackupHandler {
+  private val backupFile: String = config.backupFile
+  private val eventDispatcher: EventDispatcher = EventDispatcher(config)
```

Comment:
@deandalm I would recommend to remove `: EventDispatcher` since the type can be inferred here.